test/NuGet.Clients.Tests/NuGet.CommandLine.Test/ExtractOptionsTests.cs

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using Xunit;
+
+namespace NuGet.CommandLine.Test
+{
+    public class ExtractOptionsTests
+    {
+
+        [Fact]
+        public void ExtractOption_PassOptionNoHttpCache_NoHttpCacheShouldBeSet()
+        {
+            //setup
```

Comment:
Nit: we tend to use Arrange, Act, Assert as the three steps.
```suggestion
            // Arrange
```